mautic/mautic
# Fix broadcast query

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [X ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
We noticed the broadcast command takes a lot of time to load the records from the database this PR makes the query faster and remove any redundant queries


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Fill the database with contacts in segments
3. Run broadcast command, and record the time.
4. apply the PR and record the time again.
